assignment1

```diff
@@ -0,0 +1,46 @@
+
+//Create a java program to search through the home directory and look for files that match a regular expression. The program should be able to take inputs repeatedly and should print out the full absolute path of the matching files found.
+
+
+
+package assignments;
+
+import java.io.File;
+import java.util.Scanner;
+
+public class Assignment1 {
+    public void findFile()
+    {
+        File file=new File("/home/goutM/Desktop/git/git_katas");
+        String s;
+        int t=0;
+        Scanner sc=new Scanner(System.in);
+        while(true)
+        {
+            System.out.println("Enter the file name you want to search");
+            s=sc.nextLine();
+            String[] files=file.list();
+            for(String a:files)
+            {
+                if(a.equals(s))
+                {
+                    t=1;
+                    System.out.println("File found :" +file.getAbsolutePath());
+                    break;
+                }
+
+            }
+            if(t==0)
+            {
+                System.out.println("File not found re-enter the file name");
+            }
+            t=0;
+        }
+    }
+    public static void main(String[] args)
+    {
+        Assignment1 b=new Assignment1();
+        b.findFile();
+    }
+}
+
```

Comment:
What if the input given is a folder and not directly a filename, you are not handling that case.